### PR TITLE
Add ammo category to clips

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Someone pointed out that clips are not particularly magazines, and more fitting to be in ammo category
i agree
#### Describe the solution
Move it to ammo category
#### Describe alternatives you've considered
Some new category for clips?

### CODE REVIEW

File: data/json/items/magazine/3006.json [hunk 1]
```diff
@@ -42,6 +43,7 @@
     "//": "Previous PR, looks mostly accurate.",
     "looks_like": "stanag30",
     "type": "MAGAZINE",
+    "category": "ammo",
     "name": { "str": "M1 Garand 8-round clip" },
```
Comment (Holli-Git) [inline data/json/items/magazine/3006.json:47]: M1 Garand Clip is an en-bloc clip, it remains inside of the rifle.

File: data/json/items/magazine/66mm.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "//": "Estimated from photographs and known length of launcher.",
     "looks_like": "belt40mm",
     "type": "MAGAZINE",
+    "category": "ammo",
```
Comment (Holli-Git) [inline data/json/items/magazine/66mm.json:7]: M74 doesn't appear to allow single-loading rockets, the clip holds them all and as such is a magazine.

**Review by Holli-Git — CHANGES_REQUESTED** [review]

File: data/json/items/magazine/3006.json [hunk 3]
```diff
@@ -42,6 +43,7 @@
     "//": "Previous PR, looks mostly accurate.",
     "looks_like": "stanag30",
     "type": "MAGAZINE",
+    "category": "ammo",
```
Comment (GuardianDll) [inline data/json/items/magazine/3006.json:46]:
```suggestion
```

File: data/json/items/magazine/66mm.json (hunk 2 above)
Comment (GuardianDll) [inline data/json/items/magazine/66mm.json:7]:
```suggestion
```

**Pull request merged**